config.go

```diff
@@ -587,7 +587,7 @@ func (s *Settings) GetGithubOauthToken() (string, error) {
 
 	oauthString, err := s.GetGithubOauthString()
 	if err != nil {
-		return "", err
```

Comment:
Because we don't require this token anymore, we shouldn't return an error when a token is not set

Reply:
the TODO for this function says that we'll be removing this function, if this isn't true anymore, can we update?

Reply:
I think we can also not have this return an error anymore then; 581 could just return ""

Reply:
talked about this offline but we are switching the TODO of this ticket to DEVPROD-1429 to deprecate using github tokens